integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonExplainCommand.scala

```diff
@@ -51,8 +51,13 @@ case class CarbonExplainCommand(
       sparkSession.sessionState.executePlan(child.asInstanceOf[ExplainCommand].logicalPlan)
     try {
       ExplainCollector.setup()
-      queryExecution.toRdd.partitions
       if (ExplainCollector.enabled()) {
+        queryExecution.toRdd.partitions
+        // For count(*) queries the explain collector will be disabled, so profiler
+        // informations not required in such scenarios.
+        if(null==ExplainCollector.getFormatedOutput) {
```

Comment:
formatting is not correct please format if condition code

Reply:
update. thanks